clients/manifest.go

```diff
@@ -146,6 +146,12 @@ func GenerateAndUploadManifests(sourceManifest m3u8.MediaPlaylist, targetOSURL s
 			}
 		}
 
+		if isClip {
+			_, totalSegs := video.GetTotalDurationAndSegments(renditionPlaylist)
+			renditionPlaylist.Segments[1].Discontinuity = true
+			renditionPlaylist.Segments[totalSegs-1].Discontinuity = true
```

Comment:
Will we get an Array Out Of Bounds for a single segment clip? Possibly not because of the weird array preallocation stuff of the playlist library, but probably good to guard against anyway

Reply:
Yup exactly what we ran into. Fixed here: https://github.com/livepeer/catalyst-api/commit/dd521620011ac4e9f27840985fd4121c403026c6

Reply:
<img src="https://media1.giphy.com/media/ZGBtu79ItaUu3J587f/giphy.gif"/>